test/config.go

```diff
@@ -40,5 +41,6 @@ func NewDefaultConfig() *Config {
 		E2EImage:          "localhost:5000/pingcap/e2e-helper:latest",
 		ChaosFSImage:      "localhost:5000/pingcap/chaos-fs:latest",
 		ChaosScriptsImage: "localhost:5000/pingcap/chaos-scripts:latest",
+		ChaosDNSImage:     "pingcap/chaosdns:latest",
```

Comment:
Though the default value of them takes no effect (as they will be set by the default value of flags), keep a consistency is more tidy.

Reply:
do you mean the image should be `localhost:5000/pingcap/chaosdns:latest`?

Reply:
done